queries/deleteMovie.js

```diff
@@ -0,0 +1,10 @@
+import pool from "../database/db.js";
+
+const deleteMovie = async (id) => {
+  const deleteQuery = "delete from movies where MovieID = $1 returning *;";
+  const values = [id];
+  const res = await pool.query(deleteQuery, values);
+  return res.rows;
+};
+
+export default deleteMovie;
```

Comment:
This would be `export { deleteMovie }` if using named exports, or you can add it to the function declaration.